.circleci/mysql/env

```diff
@@ -0,0 +1,41 @@
+APP_NAME=CDash
+APP_ENV=testing
+APP_KEY=base64:/4agNXrZCKuMNyJm9lUSLXNFjASyr9xJi+mJE1ece3k=
```

Comment:
Seems like a strange thing to check in. Any security concerns here?

Reply:
Given that these files are strictly CircleCI configuration files, no, there is no concern here.

Reply:
👍